pwa/app/lib/api/AwardType.ts

```diff
@@ -0,0 +1,186 @@
+  RISING_STAR = 50,
+  CHAIRMANS_HONORABLE_MENTION = 51,
+  CONTENT_COMMUNICATION_HONORABLE_MENTION = 52,
+  TECHNICAL_EXECUTION_HONORABLE_MENTION = 53,
+  REALIZATION = 54,
+  REALIZATION_HONORABLE_MENTION = 55,
+  DESIGN_YOUR_FUTURE = 56,
+  DESIGN_YOUR_FUTURE_HONORABLE_MENTION = 57,
+  SPECIAL_RECOGNITION_CHARACTER_ANIMATION = 58,
+  HIGH_SCORE = 59,
+  TEACHER_PIONEER = 60,
+  BEST_CRAFTSMANSHIP = 61,
+  BEST_DEFENSIVE_MATCH = 62,
+  PLAY_OF_THE_DAY = 63,
+  PROGRAMMING = 64,
+  PROFESSIONALISM = 65,
+  GOLDEN_CORNDOG = 66,
+  MOST_IMPROVED_TEAM = 67,
+  WILDCARD = 68,
+  CHAIRMANS_FINALIST = 69,
+  OTHER = 70,
+  AUTONOMOUS = 71,
+  INNOVATION_CHALLENGE_SEMI_FINALIST = 72,
+  ROOKIE_GAME_CHANGER = 73,
+  SKILLS_COMPETITION_WINNER = 74,
+  SKILLS_COMPETITION_FINALIST = 75,
+  ROOKIE_DESIGN = 76,
+  ENGINEERING_DESIGN = 77,
+  DESIGNERS = 78,
+  CONCEPT = 79,
+  GAME_DESIGN_CHALLENGE_WINNER = 80,
+  GAME_DESIGN_CHALLENGE_FINALIST = 81,
+  SUSTAINABILITY = 82,
+}
+
+export const BLUE_BANNER_AWARDS = new Set([
+  AwardType.CHAIRMANS,
+  AwardType.CHAIRMANS_FINALIST,
+  AwardType.WINNER,
+  AwardType.WOODIE_FLOWERS,
+  AwardType.SKILLS_COMPETITION_WINNER,
+  AwardType.GAME_DESIGN_CHALLENGE_WINNER,
+]);
+
+export const INDIVIDUAL_AWARDS = new Set([
+  AwardType.WOODIE_FLOWERS,
+  AwardType.DEANS_LIST,
+  AwardType.VOLUNTEER,
+  AwardType.FOUNDERS,
+  AwardType.BART_KAMEN_MEMORIAL,
+  AwardType.MAKE_IT_LOUD,
+]);
+
+export const NON_JUDGED_NON_TEAM_AWARDS = new Set([
+  AwardType.HIGHEST_ROOKIE_SEED,
+  AwardType.WOODIE_FLOWERS,
+  AwardType.DEANS_LIST,
+  AwardType.VOLUNTEER,
+  AwardType.WINNER,
+  AwardType.FINALIST,
+  AwardType.WILDCARD,
+]);
+
+export const MACHINE_AWARDS = new Set([
+  AwardType.AUTONOMOUS,
+  AwardType.CREATIVITY,
+  AwardType.ENGINEERING_EXCELLENCE,
+  AwardType.INDUSTRIAL_DESIGN,
+  AwardType.INNOVATION_IN_CONTROL,
+  AwardType.QUALITY,
+]);
+
+export const TEAM_ATTRIBUTE_AWARDS = new Set([
+  AwardType.ENGINEERING_INSPIRATION,
+  AwardType.GRACIOUS_PROFESSIONALISM,
+  AwardType.IMAGERY,
+  AwardType.JUDGES,
+  AwardType.ROOKIE_ALL_STAR,
+  AwardType.ROOKIE_INSPIRATION,
+  AwardType.SPIRIT,
+  AwardType.SUSTAINABILITY,
+]);
+
+export const SUBMITTED_AWARDS = new Set([
+  AwardType.CHAIRMANS,
+  AwardType.CHAIRMANS_FINALIST,
+  AwardType.DEANS_LIST,
+  AwardType.SAFETY,
+  AwardType.WOODIE_FLOWERS,
+]);
+
+export const ROBOT_PERFORMANCE_AWARDS = new Set([
+  AwardType.FINALIST,
+  AwardType.WINNER,
+]);
+
+export enum AwardCategory {
+  MACHINE_AWARDS = 1,
+  TEAM_ATTRIBUTE_AWARDS = 2,
+  SUBMITTED_AWARDS = 3,
+  ROBOT_PERFORMANCE_AWARDS = 4,
+}
+
+export const AWARD_CATEGORIES: { [key in AwardCategory]: Set<AwardType> } = {
+  [AwardCategory.MACHINE_AWARDS]: MACHINE_AWARDS,
+  [AwardCategory.TEAM_ATTRIBUTE_AWARDS]: TEAM_ATTRIBUTE_AWARDS,
+  [AwardCategory.SUBMITTED_AWARDS]: SUBMITTED_AWARDS,
+  [AwardCategory.ROBOT_PERFORMANCE_AWARDS]: ROBOT_PERFORMANCE_AWARDS,
+};
+
+export function getNormalizedName(
```

Comment:
Is this used?

Reply:
I had this on my testing repo, and used it on the SVG banners. I didn't add the SVG banners here yet, so I'll remove this for now and add it with the banners later